Kommunicate/Classes/Extensions/KMConfiguration+Extension.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  KMConfiguration.swift
+//  Kommunicate
+//
+//  Created by Shivam Pokhriyal on 20/08/19.
+//
+
+import Foundation
+import ApplozicSwift
+
+extension ALKConfiguration {
+    public var hideFaqButtonInConversationView: Bool {
+        set {
+            guard newValue else { return }
+            navigationItemsForConversationView = []
+        }
+        get {
+            /// Return value doesn't matter
```

Comment:
@ShivamPokhriyal why is this code required if it 'doesn't matter'?

Reply:
It gives compile time error